### PR TITLE
Retry ansible-pull 10x

### PR DESCRIPTION
**Pull request by djmitche**
Hopefully this will keep our github problem under control.

We still need to solve it, though -- after all, we pull the buildbot repo quite a lot on these hosts, too.

### CODE REVIEW

File: roles/base/templates/run-ansible-pull.sh.j2 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#! /bin/sh
+#! /bin/bash
```
Comment (seankelly) [inline roles/base/templates/run-ansible-pull.sh.j2]: Bash is at `/usr/local/bin` on FreeBSD.

Comment (verm) [inline roles/base/templates/run-ansible-pull.sh.j2]: On 2016-01-16 12:10 -0800, Sean Kelly wrote:
> In roles/base/templates/run-ansible-pull.sh.j2:
> 
> > @@ -1,4 +1,4 @@
> > -#! /bin/sh
> > +#! /bin/bash
> 
> Bash is at /usr/local/bin on FreeBSD.

Is it really a bash script?


Amar.

Comment (seankelly) [inline roles/base/templates/run-ansible-pull.sh.j2]: Yes. It's using `[[ ... ]]` in two places now. But, the variables being used are integers so word-splitting isn't an issue and both use comparison operators compatible with `test`.

Comment (djmitche) [inline roles/base/templates/run-ansible-pull.sh.j2]: I think $((foo+bar)) is a bashism though?

Comment (seankelly) [inline roles/base/templates/run-ansible-pull.sh.j2]: Perhaps but it works in FreeBSD's sh.

    $ /bin/sh
    $ RETRY=0
    $ RETRY=$((RETRY+1))
    $ echo $RETRY
    1

**Comment by djmitche** [comment]
Well it doesn't hurt to run in bash.. any other issues here?

**Comment by seankelly** [comment]
LGTM

**Comment by iblis17** [comment]
Oh! This way will be more portable. :)
```shell
#! /usr/bin/env bash
```

**Comment by djmitche** [comment]
Well, portability's not a huge concern here, and easy enough to change later anyway.

**Pull request merged**